composer.json

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "ezsystems/privacy-cookie-bundle",
+    "description": "Privacy cookie banner integration bundle into eZ Publish/eZ Platform",
+    "keywords": ["cookies", "privacy", "policy"],
+    "license": "GPL-2.0",
+    "authors": [
+        {
+            "name": "Rafal Toborek",
```

Comment:
we usually credit the company and community:
https://github.com/ezsystems/ezpublish-community/blob/master/composer.json#L6